### PR TITLE
Update Clank to Ansible 2.0

### PR DESCRIPTION
**Pull request by amercer1**

### CODE REVIEW

**Comment by steve-gregory** [comment]
LG2M. 👍  @lenards any reason not to :shipit: ?

**Comment by lenards** [comment]
Looks good to me, as well. [:boat:](https://www.youtube.com/watch?v=yOWK7Tam01M) 

:shipit: @amercer1 - good work!

**Comment by lenards** [comment]
@amercer1 - I'll let you do the merge honors.

**Comment by lenards** [comment]
One thing to note here is that the duplication of tasks in `atmosphere-setup-atmosphere` & `troposphere-setup-troposphere` meant there was _more_ work to do here. Just to reinforce the notion of [Don't Repeat Yourself](http://c2.com/cgi/wiki?DontRepeatYourself).

File: roles/setup-postgres/vars/default.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+POSTGRES_SQL_INSTALL_DIRECTORY: /var/lib/postgresql 
+DB_VERSION: 9.3
+
```
Comment (lenards) [inline roles/setup-postgres/vars/default.yml]: What's the difference between `<role>/vars/default.yml` and `<role>/defaults/main.yml`? 

Are we subverting variable precedence \[1\] here? 

\[1\] http://docs.ansible.com/ansible/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

File: roles/setup-postgres/tasks/main.yml [hunk 2]
```diff
@@ -4,15 +4,13 @@
     - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
     - "{{ ansible_distribution }}.yml"
     - "{{ ansible_os_family }}.yml"
-    - "main.yml"
+    - "default.yml"
```
Comment (lenards) [inline roles/setup-postgres/tasks/main.yml]: If this remained `main.yml`, then wouldn't it match `**/*/defaults/main.yml`?

**Pull request merged**